### PR TITLE
[Merged by Bors] - chore: split RepresentationTheory.Action in multiple files

### PR DESCRIPTION
**Pull request by chrisflav**
Splits `Mathlib.RepresentationTheory.Action` in multiple files.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RepresentationTheory/Action/Limits.lean [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+Authors: Scott Morrison
+-/
+import Mathlib.CategoryTheory.Abelian.FunctorCategory
+import Mathlib.CategoryTheory.Abelian.Transfer
+import Mathlib.CategoryTheory.Linear.LinearFunctor
+import Mathlib.CategoryTheory.Linear.FunctorCategory
+import Mathlib.CategoryTheory.Preadditive.AdditiveFunctor
+import Mathlib.RepresentationTheory.Action.Basic
+
+/-!
```
Comment (jcommelin) [inline Mathlib/RepresentationTheory/Action/Limits.lean:13]: Can you please add a title for this file, like
```suggestion
/-!
# Categorical properties of `Action V G`
```
or something like that...

File: Mathlib/RepresentationTheory/Action/Monoidal.lean [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+import Mathlib.RepresentationTheory.Action.Limits
+import Mathlib.CategoryTheory.Monoidal.FunctorCategory
+import Mathlib.CategoryTheory.Monoidal.Transport
+import Mathlib.CategoryTheory.Monoidal.Rigid.OfEquivalence
+import Mathlib.CategoryTheory.Monoidal.Rigid.FunctorCategory
+import Mathlib.CategoryTheory.Monoidal.Linear
+import Mathlib.CategoryTheory.Monoidal.Braided
+import Mathlib.CategoryTheory.Monoidal.Types.Symmetric
+
+/-!
```
Comment (jcommelin) [inline Mathlib/RepresentationTheory/Action/Monoidal.lean:15]: Idem

**Review by jcommelin — COMMENTED** [review]
Thanks! This is a good file to split!

File: Mathlib/RepresentationTheory/Action/Limits.lean (hunk 1 above)
Comment (chrisflav) [reply Mathlib/RepresentationTheory/Action/Limits.lean:13]: See the latest commit.

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**